helm/idol-answerserver/Chart.yaml

```
@@ -10,7 +10,7 @@
 # END COPYRIGHT NOTICE
 apiVersion: v2
 name: idol-answerserver
-version: "0.4.0"
+version: "0.5.0"
```

```suggestion
version: "0.40"
```
Don't need to bump version again yet as we haven't published/release the 0.4.0 version